packages/desktop-client/src/components/modals/CategoryGroupMenuModal.tsx

```diff
@@ -91,7 +91,9 @@ export function CategoryGroupMenuModal({
 
   return (
     <Modal
-      title={group.name}
+      title={
+        <ModalTitle isEditable title={group.name} onTitleUpdate={onRename} />
+      }
```

Comment:
Could we remove left-padding from the mobile modal title?

<img width="487" alt="Screenshot 2024-04-06 at 19 51 10" src="https://github.com/actualbudget/actual/assets/886567/0e886229-e1a1-4193-bdb4-17ed9c085370">

Reply:
The modal title is centered. That left margin is reserved for any additional menus e.g. category modal's hide and delete menus.